Sources/Logging/Logging.swift

```diff
@@ -50,13 +80,35 @@ public struct Logger {
 }
 
 extension Logger {
+    /// Log a message passing the log level as a parameter.
+    ///
+    /// If the `logLevel` passed to this method is more severe than the `Logger`'s `logLevel`, it will be logged,
```

Comment:
Wording of "if ... is more severe ..." has to be 

> is more severe _or equal_

Since: `   if self.logLevel >= level {`